scripts/docker/install_os_dependencies.sh

```diff
@@ -19,16 +19,21 @@
 set -euo pipefail
 
 if [[ "$#" != 1 ]]; then
-    echo "ERROR! There should be 'runtime' or 'dev' parameter passed as argument.".
+    echo "ERROR! There should be 'runtime', 'ci' or 'dev' parameter passed as argument.".
     exit 1
 fi
 
+AIRFLOW_PYTHON_VERSION=${AIRFLOW_PYTHON_VERSION:-v3.10.10}
```

Comment:
We should add "install_os_dependencise.sh" to the upggrade mechanism (maybe with new regexp) in "update_installers_and_precommit.sh" - BTW I think we should also rename that change to something more generic - for example `upgrade_system_dependencies.sh` ?